### DISCUSSION TITLE
Merchant not showing in Datafeedr API

### DISCUSSION THREAD

**Discussion by rhsoon1** [datafeedr/wordpress-plugins#9] [Help]
Hi. I joined 8 programs in Sharesale. All of them provide data feed in their merchant detail. I ,however, only see 3 of them in datafeedr API merchants section.  Does that mean I cannot use their datafeed that don't show in datafeedr API?
Thanks

**Comment by stev1972** [comment]
If you send us your list of missing Sharesale merchants we can verify why these merchants are not available.

Please use our [contact form](https://datafeedr.me/contact) to send us your list.
